packages/theme/components/AppHeader.vue

```diff
@@ -183,16 +182,25 @@ export default defineComponent({
 
     useFetch(async () => {
       await categoriesListLoad({ pageSize: 20 });
-      categoryTree.value = categoryGetters
-        .getCategoryTree(categoryList.value?.[0])
-        ?.items.filter((c) => c.count > 0);
+
+      function prepareMenuData() {
+        return categoryList.value?.[0]?.children
+          .map((category) => ({
+            includeInMenu: category.include_in_menu,
+            label: category.name,
+            slug: `/${category.url_path}${category.url_suffix}`,
+          }))
+          .filter((category) => category.includeInMenu) ?? [];
```

Comment:
```suggestion
          .filter((category) => category.include_in_menu) 
          .map((category) => ({
            includeInMenu: category.include_in_menu,
            label: category.name,
            slug: `/${category.url_path}${category.url_suffix}`,
          })) ?? [];
```

Nitpick but feels weird to map an element only to filter it out later. 
You can also rework map and filter into a single reduce if you want